controllers/dspipeline_params.go

```diff
@@ -630,6 +632,57 @@ func (p *DSPAParams) ExtractParams(ctx context.Context, dsp *dspa.DataSciencePip
 			}
 		}
 
+		if cfg := p.APIServer.CustomKfpLauncherConfig; cfg != nil && cfg.Name != "" {
+			if cm, err := util.GetConfigMap(ctx, cfg.Name, p.Namespace, client); err != nil {
+				// If the custom kfp-launcher configmap is not available, that is OK
+				if !apierrs.IsNotFound(err) {
+					log.Info(fmt.Sprintf("Encountered error when attempting to fetch ConfigMap: [%s], Error: %v", cfg.Name, err))
+					return err
+				}
```

Comment:
looks like the comment is saying "that is OK" but we're report an error here, so it seems that it is not Okay, and we should state as such